src/sql/workbench/browser/designer/designer.ts

```diff
@@ -983,7 +983,6 @@ export class Designer extends Disposable implements IThemable {
 				table.onBlur((e) => {
 					currentTableActions.forEach(a => a.updateState());
 					table.grid.setSelectedRows([]);
-					table.grid.resetActiveCell();
```

Comment:
Is this the line causing the focus issue of properties window?

If so, directly removing this line would cause the regression you mentioned in previous PR right?
![image](https://user-images.githubusercontent.com/21186993/168189492-41f6a5b3-bdc8-497e-92a9-e14a7893a321.png)

https://github.com/microsoft/azuredatastudio/pull/19154/files#r870335645

Reply:
Also, if the fix is not trivial, I'd recommend creating a separate issue and PR for the fix.